pkg/controller/statefulset/stateful_set_utils.go

```diff
@@ -219,6 +219,10 @@ func isTerminating(pod *v1.Pod) bool {
 	return pod.DeletionTimestamp != nil
 }
 
+// isPending returns true if pod has a Phase of PodPending
+func isPending(pod *v1.Pod) bool{
+	return pod.Status.Phase != v1.PodPending
```

Comment:
The func name is `isPending` and your check is `pod.Status.Phase != v1.PodPending`. Is it expected?

Reply:
My fault, i have corrected it, THX!